ovos_persona_server/files.py

```
@@ -0,0 +1,355 @@
+    storage_dir = os.path.abspath(settings.file_storage_path)
+    os.makedirs(storage_dir, exist_ok=True)
+    return storage_dir
+
+
+# --- Files Endpoints ---
+
+@files_router.post("/", response_model=FileObject, status_code=status.HTTP_200_OK)
+async def upload_file(
+        file: UploadFile = File(..., description="The file to upload."),
+        purpose: FilePurpose = Form(...,
+                                    description="The intended purpose of the file. Currently only 'assistants' is supported."),
```

Ruff (0.14.0)

Do not perform function call `Form` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)